### PR TITLE
rfp js post

### PR DESCRIPTION
**Pull request by Athanasiamo**

### CODE REVIEW

File: content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+title: Request for Proposal - Javascript Development
```
Comment (maelle) [inline content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd:2]:
```suggestion
title: Request for Proposal - Javascript Development (Contract Work)
```

Not sure if useful though

File: content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+description: ''
+comment: no
+toc: no
+autoCollapseToc: no
+contentCopyright: no
+reward: no
+mathjax: no
+---
+
+**R-Ladies wants to implement some new functionalities (using Javascript) for a re-implementation of their  website, whose source code may be found on [GitHub](https://github.com/rladies/website).**
```
Comment (maelle) [inline content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd:19]:
```suggestion
**The R-Ladies global organization wants to implement some new functionalities (using Javascript) for a re-implementation of their  website, whose source code may be found on [GitHub](https://github.com/rladies/website).**
```

File: content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+## Company Background
+R-Ladies, is a worldwide organisation whose mission is to promote gender diversity in the R community. 
+
+The R community suffers from an underrepresentation of minority genders (including but not limited to cis/trans women, trans men, non-binary, genderqueer, agender) in every role and area of participation, whether as leaders, package developers, conference speakers, conference participants, educators, or users (see recent stats).
+
+As a diversity initiative, the mission of R-Ladies is to achieve proportionate representation by encouraging, inspiring, and empowering people of genders currently underrepresented in the R community. R-Ladies’ primary focus, therefore, is on supporting minority gender R enthusiasts to achieve their programming potential, by building a collaborative global network of R leaders, mentors, learners, and developers to facilitate individual and collective progress worldwide.
+
+
+### Project Manager
+Athanasia Monika Mowinckel
```
Comment (maelle) [inline content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd:36]: Add a link to your website?

File: content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+
+## Budget
+We ask prospective contractors to provide a preliminary budget for their proposal. This can be provided as hourly rates with estimated amounts of hours to complete the project. Development work can be unpredictable, and we understand that estimated hours to completion might diverge somewhat from proposal estimates. 
+
+## Timeline
+
+### RFP Release Date
+The date you’ve sent out the RFP. This lets vendors see how recent the request is.
```
Comment (maelle) [inline content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd:45]: I thought vendor = people writing the code and R-Ladies = customer?

**Review by maelle — COMMENTED** [review]
:rocket:

File: content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd (hunk 4 above)
Comment (Athanasiamo) [reply content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd:45]: I'm going to delete this part, I forgot to when I copy pasted! The release date is the date the blogpost goes live :)

Comment (Athanasiamo) [reply content/post/2022-03-28-request-for-proposal-javascript-development/index.en.Rmd:45]: but yes, vendor is people writing the code, and we are the customer. So I think the intent here is to have a date that shows when the request was launched, so vendors can assess if this is a recent or old request.

**Pull request merged**